doc/source/indexing.rst

```diff
@@ -1689,6 +1689,12 @@ faster, and allows one to index *both* axes if so desired.
 Why does assignment fail when using chained indexing?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+.. note::
+
+   The chained assignment warnings / exceptions are aiming to inform the user of a possibly invalid
+   assignment. There may be false positives; situations where a chained assignment is inadvertently
```

Comment:
remove the last sentence; this is actually not true. All purported false positive are actually user error (at least that have been reported)